exercises/joseph_ramsay/ex01_basics/src/jabberwock.txt

```diff
@@ -0,0 +1,34 @@
+’Twas brillig, and the slithy toves
```

Comment:
This file should be attached to the PR and not part of the source code. It is only for the reviewer to test your changes.